api/src/__tests__/conversationMutations.test.ts

```diff
@@ -903,4 +907,94 @@ describe('Conversation message mutations', () => {
       fail('Conversation not found to update operator status');
     }
   });
+
+  test('Convert conversation to card', async () => {
+    const conversation = await conversationFactory({
+      assignedUserId: user._id
+    });
+    const stage = await stageFactory({ type: 'deal' });
+
+    const mutation = `
+      mutation conversationConvertToCard($_id: String!, $type: String!, $itemId: String, $itemName: String, $stageId: String) {
```

Comment:
instantiate it into a whole test scope variable & call it twice instead of duplicating them